### DISCUSSION TITLE
Rule BR-BT-00760-0036

### DISCUSSION THREAD

**Discussion by StephaneElGuedj** [OP-TED/eForms-SDK#1025] [Q&A] [answered]
BR-BT-00760-0036 rule states that: 
either BT-760-LotResult should be filled, or BT-142-LotResult = 'open-nw' (no winner, competition is still ongoing).

Shouldn't it be:

either BT-760-LotResult should be filled, or BT-142-LotResult = 'open-nw' or BT-144-LotResult = 'no-rece' (No tenders, requests to participate or projects were received) ?

Users don't understand that they have to fill BT-760-LotResult whereas they already indicated that no tenders, requests to participate or projects were received...

Will the rule expression be updated in the future (at least in SDK 1.13) ?

**Comment by YvesJo** [comment, marked as answer]
Hi,
This may look like redundant information. However, in the regulation Annex, the BTs have been identified as mandatory (M) and not as conditionally mandatory (CM) and this should most likely be reviewed. 
Some other reasons for non-award could however still be source of uncertainty.
It is not foreseen to have such a change as part of SDK 1.13 as some internal clarifications would be needed before applying such a change.

**Reply by StephaneElGuedj** [reply]
OK. Thanks for your input.
